index.html

```diff
@@ -14,6 +14,14 @@
     <script src="https://kit.fontawesome.com/86cc835e69.js" crossorigin="anonymous"></script>
     <link rel="shortcut icon" href="assets/bw.png">
     <link rel="stylesheet" href="styles.css">
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-YCMK4PVT08"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'G-YCMK4PVT08');
+    </script>
```

Comment:
Hi broth! very good job.
This is ok to merge.

<img src="https://media3.giphy.com/media/RLW9YEaSBfBMt79fm4/giphy.gif"/>